pkgs/by-name/tw/twitch-dl/package.nix

```diff
@@ -38,6 +39,7 @@ python3Packages.buildPythonApplication rec {
 
   nativeCheckInputs = [
     python3Packages.pytestCheckHook
+    writableTmpDirAsHomeHook
```

Comment:
This should go in nativeInstallCheckInputs, to make it exactly equivalent to the previous version.

Reply:
Oh right, Python packages have their phases ordered differently.. nevermind then.